driver/esp32/ili9XXX.py

```diff
@@ -429,30 +438,35 @@ def flush(self, disp_drv, area, color_p):
         # esp.spi_device_acquire_bus(self.spi, esp.ESP.MAX_DELAY)
 
         # Column addresses
+        self.send_cmd(0x2A)
 
-        self.send_cmd(0x2A);
+        x1 = area.x1 + self.start_x if self.start_x else area.x1
+        x2 = area.x2 + self.start_x if self.start_x else area.x2
```

Comment:
Why not set `self.start_x/y` to 0 by default, and always add `area.x/y + self.start_x/y`?
if statements are more expensive

Reply:
That makes sense. Will do.